experiments/snippets/autoLoadMore/README.md

```diff
@@ -0,0 +1,6 @@
+# autoLoadMore.js
+This inline javascript (NOT snippet) used to load more rows into a model when the "Load More" button is scrolled into the window.  This creates a scroll and load experience like salesforce.  There must be enough rows in the model to allow for a scroll though - I'm sure other trigger events could be provided, but this is based on too many to load at once in a performance friendly experience, and then load more when scrolling long enough.  We tend to use this with the "Max # of records (Limit)" field set between 50-100, but it will dependend on what is being queried and when you see a performance degredation from overloading.  It will require modification of the model and field names to match yours.
+
+
+# autoLoadMore.xml
+This is a sample page wich uses this inline javascript to load more records.  In a standard developer org, there are likely not enough sample accounts to really leverage this, so the "Max # of records (Limit)" is set very low.  You'll need to zoom the page so you have to scroll to see the last row.  
```

Comment:
Copy suggestions + add the suggestion to resize browser window (which I found easier than zooming in a lot)
```suggestion
This is a sample page which uses this inline JavaScript to load more records. In a standard developer org, there are likely not enough sample accounts to really leverage this, so the "Max # of records (Limit)" is set very low.  You'll need to zoom the page or decrease the size of your browser window so you have to scroll to see the last row.  
```